src/sentry/integrations/vsts/client.py

```diff
@@ -132,6 +134,30 @@ def get_work_item(self, instance, id):
             ),
         )
 
+    def get_work_item_states(self, instance, project=None):
+        if project is None:
+            # TODO(lb): I'm pulling from the first project.
+            # Not sure what else to do here unless I can prompt the user
+            project = self.get_projects(instance)['value'][0]['id']
```

Comment:
The statuses are not standard and can be changed by the user on their end. Really I need the VSTS project to ensure I'm getting exactly what I need here. I just don't have enough insight at this time in the react to:
1. Choose a project
2. Then and only then show the status possibilities. 

It may need to be addressed. I am currently just getting the first project.

Reply:
Sound's like we'll probably want to address this laster to support reloading the form fields